update/export jump interest

khanhhaquang opened the pull request:
## Description
- Exclude `integrationTests` for Prod build
- Export `jumpInterestV2` to index
- Export minor demo for `@konomi-network/client` package

https://www.npmjs.com/package/@konomi-network/client

khanhhaquang committed:
export jumpInterst

khanhhaquang committed:
Publish

 - @konomi-network/decentralized-fs@1.0.2-alpha.0
 - @konomi-network/eslint-config-konomi@1.0.3-alpha.0
 - konomi-cli@1.0.1-alpha.0
 - @konomi-network/client@0.1.0

khanhhaquang committed:
update private

khanhhaquang committed:
Publish

 - konomi-cli@1.0.2-alpha.0
 - @konomi-network/client@0.2.0

khanhhaquang committed:
update cli repo name

khanhhaquang committed:
update path

khanhhaquang committed:
update path

khanhhaquang committed:
Publish

 - @konomi-network/cli@1.0.3-alpha.0
 - @konomi-network/client@0.3.0

khanhhaquang committed:
update tsconfig prod

khanhhaquang committed:
Publish

 - @konomi-network/client@0.4.0

khanhhaquang committed:
update README

khanhhaquang committed:
Publish

 - @konomi-network/client@0.5.0

khanhhaquang committed:
comment logger

khanhhaquang committed:
Publish

 - @konomi-network/client@0.5.1

lispking commented:
I changed the feature/KON-525 branch to the mocha method. There is a lot of code here that conflicts with my branch. I think the original writing method is not appropriate. It should be done with a testing framework.

khanhhaquang committed:
update config path

khanhhaquang committed:
solve conflict

khanhhaquang committed:
update eslint config

khanhhaquang committed:
export Comptroller and OToken

khanhhaquang committed:
Publish

 - @konomi-network/client@0.5.2

khanhhaquang committed:
fix lint

khanhhaquang committed:
fix lint

khanhhaquang committed:
ignore test in prod

khanhhaquang committed:
Merge branch 'ocean-lending' of github.com:konomi-network/typescript-common into refactor/export-jumpInterest

khanhhaquang committed:
Publish

 - @konomi-network/decentralized-fs@1.0.2
 - @konomi-network/eslint-config-konomi@1.0.4-alpha.0
 - @konomi-network/client@0.5.3

khanhhaquang committed:
ignore dist

khanhhaquang committed:
Publish

 - @konomi-network/client@0.5.4

khanhhaquang committed:
ignore in prod build script

khanhhaquang committed:
Publish

 - @konomi-network/decentralized-fs@1.0.3
 - @konomi-network/client@0.5.5

khanhhaquang committed:
Merge branch 'ocean-lending' of github.com:konomi-network/typescript-common into refactor/export-jumpInterest

khanhhaquang committed:
solve conflict

khanhhaquang committed:
move exporting client

khanhhaquang committed:
Publish

 - @konomi-network/client@0.6.0

khanhhaquang committed:
correct syntax
